src/commands/init.ts

```diff
@@ -1,56 +1,68 @@
-import { Command, Confirm, Input, List, yellow } from "../../deps.ts";
-import { Config, ConfigFormats, parseConfig, writeConfig } from "../config.ts";
-import { configExists, pathExists } from "../utilities/files.ts";
+import { Command, Confirm, Input, List, log, Select } from "../../deps.ts";
+import {
+  Config,
+  ConfigFormat,
+  configFormat,
+  defaultConfig,
+  readConfig,
+  writeConfig,
+} from "../config.ts";
 import { version } from "../version.ts";
 
-function detectConfig(): ConfigFormats {
-  if (pathExists("egg.yaml")) return "yaml";
-  else if (pathExists("egg.yml")) return "yml";
-  return "json";
+/**
```

Comment:
As we try to follow Deno style guide, this includes
[Vertical spacing should be minimized whenever possible.](https://github.com/denoland/deno/blob/4ebd24342368adbb99582b87dc6c4b8cb6f44c87/docs/contributing/style_guide.md#use-jsdoc-for-exported-symbols)
```ts
/** Init Command.
 * `init` creates (or overrides) configuration in
 * the cwd with an interactive prompt. */
```
The same goes for the other comments

Reply:
:)